src/ui/server/helpers/summary-lists/your-business/nature-of-your-business-fields/index.ts

```diff
@@ -47,7 +51,10 @@ const generateNatureOfYourBusinessFields = (answers: ApplicationBusiness, refere
     }),
   ] as Array<SummaryListItemData>;
 
-  return fields;
+  return {
+    title: FORM_TITLE,
```

Comment:
we're doing this in a few places.  Should we have a helper for this and return the helper?

Reply:
What would the helper do exactly? I think a helper would overcomplicate it. Seems we'd be calling a function with a `FORM_TITLE` and the helper would just return an object with that `FORM_TITLE`?